docs/release-notes/2.32.0.md

```diff
@@ -0,0 +1,167 @@
+
+| **Release date:** | Nov 7, 2022                                                                                    |
+| ----------------- | ----------------------------------------------------------------------------------------------- |
+| **Installation:** | [Installing Percona Monitoring and Management](https://www.percona.com/software/pmm/quickstart) |
+
+Percona Monitoring and Management (PMM) is an open source database monitoring, management, and observability solution for MySQL, PostgreSQL, and MongoDB.
+
+!!! caution alert alert-warning "Important"
+    We recommend using the latest version of PMM. This ensures that you have access to the latest PMM features and that your environment runs on the latest version of the underlying components, such as VictoriaMetrics, with all the bug fixes in place.
+
+## Release Highlights
+
+### MongoDB Backup Management is now generally available
+
+We are happy to announce that the Backup Management for MongoDB has reached General Availability and is no longer in Technical Preview.
+
+This change comes with significant functionality enhancements and added UX improvements:
+
+- **Point-in-time recovery (PITR) restores for MongoDB backups**: This enables you to restore Amazon S3 backups from any checkpoint in the past.
+- **Local client storage**: In addition to Amazon S3, we have now also enabled the **Local Client** location option, which saves backups on the remote folder mounted to PMM client host.  
+- **Key UX improvements**: You can now create on-demand and scheduled backups from the same window instead of using separate modals.
+    In addition, we have added more icons and renamed tabs columns to better label the actions available behind important options.
+
+#### Supported setups
+
+MongoDB Backup Management now supports replica set setups for the following actions:
+
+- Create logical snapshot backups
+- Create logical PITR backups
+- Create physical snapshot backups. This is available only with Percona Server for MongoDB.
+- Restore logical snapshot backups
+- Restore physical backups. This requires additional manual operations after the restore and is only available with Percona Server for MongoDB.
+- Restore logical PITR backups from S3.
+  
+#### Current limitations
+
+- Restoring logical PITR backups only supports S3 storage type.
+- Restoring physical backups requires manual post-restore actions.
+- Restoring a MongoDB backup on a new cluster is not yet supported.
+- Restoring physical backups for containerized MongoDB setups is not supported.
+- Local storage for MySQL is not supported.
+- Sharded cluster setups are not supported.
+- Backups that are stored locally cannot be removed automatically.
+- Retention is not supported for PITR artifacts.
+
+#### What’s next?
+
+As you can well guess, our immediate plan for Backup Management is to continue to iterate on the current limitations listed above and make the restore processes as seamless as possible for all database types.
+
```

Comment:
```suggestion

```

Let's keep this for the release blog post

Reply:
@catalinaadam Please take care of this...